src/main/scala/org/ergoplatform/nodeView/wallet/Registry.scala

```diff
@@ -0,0 +1,138 @@
+package org.ergoplatform.nodeView.wallet
+
+import io.iohk.iodb.ByteArrayWrapper
+import org.ergoplatform.nodeView.history.ErgoHistory.Height
+
+import scala.collection.mutable
+
+class Registry {
```

Comment:
We can remove all the synchronisation from Registry if we plan to have it only as private Actor field.

Otherwise we need to provide additional synchronisation for mutable field reads, not only for writes.